### DISCUSSION TITLE
The `searchParameters` option expects an object.

### DISCUSSION THREAD

**Discussion by michaellaw1701** [algolia/instantsearch#6140] [Q&A]
Hi,

We had a problem with Algolia this morning as we rebuilt our packages and it broke.

This was the error: Error: The `searchParameters` option expects an object. 

We tracked it down to updating "algoliasearch": "^4.13.1" and "vue-instantsearch": "^4.3.3" so we had to rollback to a previous lock file to get the correct Algolia packages and dependencies for search to work.

**Comment by dhayab** [comment]
Hi, what were your previous versions of both those libraries? The error most likely comes from here https://github.com/algolia/instantsearch/blob/master/packages/instantsearch.js/src/connectors/configure/connectConfigure.ts#L86, and it has been like this for 5 years already.

Don't hesitate to provide more info and a sandbox so we can look more into this.

**Reply by michaellaw1701** [reply]
It is weird as when I do an update now it all works. My guess is there was an issue with Algolia yesterday which has been fixed :)
